pkg/antctl/command_definition_test.go

```diff
@@ -38,8 +38,10 @@ import (
 	"antrea.io/antrea/pkg/antctl/transform/common"
 	"antrea.io/antrea/pkg/antctl/transform/controllerinfo"
 	"antrea.io/antrea/pkg/antctl/transform/networkpolicy"
+	"antrea.io/antrea/pkg/apis/controlplane/v1beta2"
 	cpv1beta "antrea.io/antrea/pkg/apis/controlplane/v1beta2"
 	"antrea.io/antrea/pkg/apis/crd/v1beta1"
+	network_policy "antrea.io/antrea/pkg/controller/networkpolicy"
```

Comment:
Better to name it `controllernetworkpolicy`, it's not common to rename a package with `_`.

Reply:
Fixed